algorithms/persian.sbl

```diff
@@ -55,11 +54,11 @@ externals ( stem )
 define Normalize as (
     do repeat (
         [substring] among (
-            '{U+0643}' ( <- '{kaf}' ) // Normalize Kaf
+            '{U+0643}' ( <- '{kaf}' )             // Normalize Kaf
             '{U+0626}' '{U+064A}' ( <- '{ye}'  )  // Normalize Ye
-            '{U+0629}' ( <- '{he}'  )  // Normalize Heh
+            '{U+0629}' ( <- '{he}'  )             // Normalize Heh
             '{U+0623}' '{U+0625}' ( <- '{alef}' ) // Normalize Alef
-            '{U+0624}' ( <- '{vav}' )  // Normalize Waw and Waw with Hamza above
+            '{U+0624}' ( <- '{vav}' )             // Normalize Waw and Waw with Hamza above
```

Comment:
As I noted before, it'd be better to use stringdef names for everything.

Then it's clearer to the reader what the code is doing.  Consider the line:

```
            '{U+0623}' '{U+0625}' ( <- '{alef}' ) // Normalize Alef
```

Currently I need to e.g. look up U+0623 to find it's "ARABIC LETTER ALEF WITH HAMZA ABOVE", then look up U+0625 to find it's "ARABIC LETTER ALEF WITH HAMZA BELOW" before I can fully understand this single line.  If it instead read something like this then it would be clear without additional research:

```
            '{alef_with_hamza_above}' '{alef_with_hamza_below}' ( <- '{alef}' ) // Normalize Alef
```